src/inputs/WizSingleSelect.tsx

```diff
@@ -74,9 +80,51 @@ export function WizSingleSelect(props: WizSingleSelectProps) {
         )
     }
 
+    const helperTextInvalid = (error: string | undefined) => (
+        <Split>
+            <SplitItem isFilled>
+                <span className="pf-c-form__helper-text pf-m-error">{error}</span>
+            </SplitItem>
+            <SplitItem>
+                {prompt?.label && prompt?.href && (
+                    <Button
+                        variant="link"
+                        style={{ paddingRight: '0px' }}
+                        onClick={() => window.open(prompt.href)}
+                        isDisabled={prompt?.isDisabled}
+                    >
+                        {prompt.label}
+                        <ExternalLinkAltIcon style={{ verticalAlign: '-0.125em', marginLeft: '8px' }} />
+                    </Button>
+                )}
+            </SplitItem>
+        </Split>
+    )
+
+    const helperTextPrompt = (
+        <Split>
+            <SplitItem isFilled>
+                <span className="pf-c-form__helper-text">{helperText}</span>
+            </SplitItem>
+            <SplitItem>
+                {prompt?.label && prompt?.href && (
+                    <Button
+                        variant="link"
+                        style={{ paddingRight: '0px' }}
+                        onClick={() => window.open(prompt.href)}
+                        isDisabled={prompt?.isDisabled}
+                    >
+                        {prompt.label}
+                        <ExternalLinkAltIcon style={{ verticalAlign: '-0.125em', marginLeft: '8px' }} />
+                    </Button>
+                )}
+            </SplitItem>
+        </Split>
+    )
+
```

Comment:
- This code seems highly duplicative.
- Please look at the `icon` and `iconPosition` props on `Button` instead of using hard-coded spacing. (This will also avoid the link underline extending through the space between the text and icon.)